include/rppt_tensor_color_augmentations.h

```diff
@@ -71,7 +71,8 @@ RppStatus rppt_gamma_correction_gpu(RppPtr_t srcPtr, RpptDescPtr srcDescPtr, Rpp
 /******************** blend ********************/
 
 // Alpha blending augmentation for a NCHW/NHWC layout tensor
-// *param[in] srcPtr source tensor memory
+// *param[in] srcPtr1 source tensor memory
```

Comment:
Please update the documentation as 
// *param[in] srcPtr1 source1 tensor memory
// *param[in] srcPtr2 source2 tensor memory

Reply:
Sure, I had pulled in master thus this issue came up.
Will fix it.